test/src/main/java/org/apache/accumulo/test/metrics/TestStatsDSink.java

```diff
@@ -102,7 +103,8 @@ public TestStatsDSink() throws SocketException {
         DatagramPacket packet = new DatagramPacket(buf, len);
         try {
           sock.receive(packet);
-          received.add(new String(packet.getData()));
+          byte[] trimmedBuffer = Arrays.copyOf(packet.getData(), packet.getLength());
+          received.add(new String(trimmedBuffer));
```

Comment:
Could this use the String constructor that takes start and length, something like 
```
          received.add(new String(packet.getData(),0, packet.getLength()));
```
That would avoid an extra copy operation